### PR TITLE
Update installing_cli.mdx

### PR DESCRIPTION
**Pull request by smaher-edb**
## What Changed?

- Fixed typo

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

**Comment by smaher-edb** [comment]
Also, I will request that please move following section from `using_cli.mdx` to the end of `installing_cli.mdx` file. This section is important for the manual installation option. And needs to be put at some place where it will get noticed. In `using_cli.mdx` it is kind of getting hidden.

```
Requirements
The PGD CLI requires postgres superuser privileges to run.
```

**Comment by smaher-edb** [comment]
If we are creating new page for requirement then perhaps following section from `index.mdx` can also be moved there. It is also kind of requirement that one can run PGD CLI only against BDR 4.1 and later.

```
Supported version

The PGD CLI supports BDR 4.1 and later.
```

But anyway it LGTM.

File: product_docs/docs/pgd_cli/1/installing_cli.mdx [hunk 1]
```diff
@@ -6,7 +6,7 @@ navTitle: "Installing the PGD CLI"
 
 ## Installing with TPAexec
 
-If you are using TPAexec to deploy EDB Postgres Distributed and the default deployment installs the PGD CLI package (`edb-pgd-cli`) on each BDR node automatically. It also creates the configuration file (`/etc/edb/pgd-config.yml`) prepopulated with the endpoints of each node in the cluster so no additional configuration is necessary.
+If you are using TPAexec to deploy EDB Postgres Distributed, it installs the PGD CLI package (`edb-pgd-cli`) on each BDR node, by default. It also creates the configuration file (`/etc/edb/pgd-config.yml`) prepopulated with the endpoints of each node in the cluster so no additional configuration is necessary.
```
Comment (smaher-edb) [inline product_docs/docs/pgd_cli/1/installing_cli.mdx:9]: Sorry, forgot to mention earlier. Can we add TPAexec version (v22.14) which supports PGD CLI? Perhaps something like - 
```If you are using TPAexec (v22.14 and later) to deploy EDB Postgres Distributed... ```

TPAexec v22.14 is the latest version released last week along with PGD CLI which supports it.

**Review by drothery-edb — APPROVED** [review]
Want me to merge it for you?

**Comment by drothery-edb** [comment]
@smaher-edb , I'm going to merge this so we can get it into today's build.

**Pull request merged**